RadixWallet/Features/GatewaySettingsFeature/Components/GatewayRow/GatewayRow+Reducer.swift

```diff
@@ -3,11 +3,14 @@ import SwiftUI
 
 // MARK: - GatewayRow
 public struct GatewayRow: Sendable, FeatureReducer {
+	@ObservableState
```

Comment:
Why make this Observable if we still use the old ViewStore paradigm? We should ideally migrate to observable btw.

Reply:
Aha, it's a different reducer